library/lms.c

```diff
@@ -65,7 +65,6 @@
 
 /* Currently only support H=10 */
 #define H_TREE_HEIGHT_MAX                  10
-#define MERKLE_TREE_NODE_AM_MAX            (1u << (H_TREE_HEIGHT_MAX + 1u))
 #define MERKLE_TREE_NODE_AM(type)          (1u << (MBEDTLS_LMS_H_TREE_HEIGHT(type) + 1u))
```

Comment:
MSVC is unhappy, see https://github.com/Mbed-TLS/mbedtls/pull/6584#discussion_r1023147349

Reply:
Reading MSVC suggestion on their website they want the whole thing to be 64 bit in that case I think.

Took a stab. Let's see if that silences it, otherwise will try the size_t cast.

Thanks for your help!

Reply:
This looks different, any chance this is unrelated?

Reply:
It was the same error message, and your fix made it pass, so all good here.